packages/table/src/headers/columnHeaderCell.tsx

```diff
@@ -98,6 +102,13 @@ export class ColumnHeaderCell extends AbstractComponent<IColumnHeaderCellProps,
             || target.classList.contains(Classes.TABLE_HEADER_CONTENT);
     }
 
+    public constructor() {
+        super();
+        this.state = {
+            isActive: false,
+        };
+    }
```

Comment:
❌ don't define constructor for static state, just do `public state = { isActive: false };` and be done with it. constructor is only needed if initial state depends on props.